proxy/src/http/project.rs

```diff
@@ -59,6 +60,7 @@ where
     http::with_context(ctx)
         .and(warp::get())
         .and(document::param::<String>("id", "Project id"))
+        .and(path::end())
```

Comment:
What's this? :thinking:

Reply:
idk i think it snuck in during a merge, will remove it if it's not necessary

Reply:
I'd have to check. Not sure what it's for. Can you check the docs and see if it's necessary?

Reply:
Yeah I'll take a look at the docs and try to summarize findings. If everything works & the tests pass without it I'll get rid of it

Reply:
It seems like it's just more specific about where exactly the `path` should end... this endpoint functions properly with or without it. @xla any feelings about using `end`? From the [docs](https://docs.rs/warp/0.1.22/warp/filters/path/fn.end.html): 

> Note that not including end() may result in shorter paths like /math unintentionally matching /math/sum.

Tbh I'm fine with either; I see we're using `path::end` in some other places so we should pick a convention and stick with it.

Reply:
The `end` is significant, as it prevents matches in situations where you have a route for `/` and a route for `/foo/`. The latter might never be triggered as the first one consumes the path until the first slash. So if we want an explicit match on `GET /` we should always add the `/`.

Reply:
Ah cool, thanks for enlightening me. Damn ambiguous grammars :)